ci/rpm/scan_daos_node.sh

```diff
@@ -14,6 +14,18 @@ pushd "/var/tmp/${lmd_src}"
 sudo ./install.sh
   sudo ln -s /usr/local/maldetect/ /bin/maldet
 popd
+
+fc_conf="/etc/freshclam.conf"
+sudo chmod 766 "$fc_conf"
+if ! grep -q 'ScriptedUpdates no' "$fc_conf"; then
+  printf "ScriptedUpdates no\n" >> "$fc_conf"
+fi
+: "${JOB_URL:=${JENKINS_URL}job/clamav_daily_update/}"
+if ! grep -q "$JENKINS_URL" "$fc_conf"; then
+  printf "PrivateMirror %s\n" \
+         "${JOB_URL}lastSuccessfulBuild/artifact/download/clam" >> "$fc_conf"
+fi
+sudo chmod 744 "$fc_conf"
```

Comment:
```suggestion

fc_conf="/etc/freshclam.conf"
if ! grep -q 'ScriptedUpdates no' "$fc_conf"; then
  sudo bash -c "echo \"ScriptedUpdates no\" >> \"$fc_conf\""
fi

: "${JOB_URL:=${JENKINS_URL}job/clamav_daily_update/}"
if ! grep -q "$JENKINS_URL" "$fc_conf"; then
  sudo bash -c "echo \"PrivateMirror ${JOB_URL}lastSuccessfulBuild/artifact/download/clam\" >> \"$fc_conf\""
fi
```

FWIW, `printf` is overkill for just adding a line with no formatting.

Reply:
Will be in the next push